foundations/introduction/join_the_odin_community.md

```diff
@@ -84,7 +84,11 @@ Your Multiple Lines of Colorful Code
 
   1. First, create a free [GitHub account](https://github.com/join). As you will discover, GitHub is an integral part of the development workflow.
 
-  2. In a moment we're going to ask you to join the Discord community, here are some guidelines before you dive in: 
+  2. Now, Sign in to our [Discord server](https://discord.gg/fbFCkYabZB). Pop in and say hello! We've created an introductions room which is a great place to introduce yourself and we're always happy to welcome new community members. We have chat rooms for every development topic covered in our curriculum. Log into the chat and start exploring!
```

Comment:
Nitpick: "Sign" should be lowercase here

Reply:
Sure. Thank you. I'll update it.